source/common/stats/thread_local_store.cc

```diff
@@ -496,6 +506,39 @@ Histogram& ThreadLocalStoreImpl::ScopeImpl::histogramFromStatName(StatName name)
   return **central_ref;
 }
 
+absl::optional<std::reference_wrapper<const Counter>>
+ThreadLocalStoreImpl::ScopeImpl::findCounter(StatName name) const {
+  if (parent_.rejectsAll()) {
```

Comment:
Do you have coverage for rejection here and below? Is it worth it to even check in the find case?

Reply:
Good point; IMO it's not necessary to do the early reject exit here; that was done in other cases to avoid some memory allocation and/or taking a lock, which don't apply here.

Reply:
Removed early rejects.